### PR TITLE
Add the test host + docs.

### PR DESCRIPTION
**Pull request by jverkoey**
This is required because `@build_bazel_rules_apple//apple/testing/default_host/ios` has gone away as of https://github.com/bazelbuild/rules_apple/commit/0b130b4c675f0de6e61d5f5112c12ae7e0ed6f66#diff-41a7d742e7bd3a31012126217229e33b

Part of https://github.com/material-components/material-components-ios/issues/9363

### CODE REVIEW

File: .bazelrc [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2019-present The Material Foundation Authors. All Rights Reserved.
```
Comment (romoore) [inline .bazelrc]: Date?

Comment (jverkoey) [reply .bazelrc]: Fixed.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,67 @@
+bazel-*
+.kokoro-ios-runner
+.clang-format-ci-src
+
+# Xcode
+#
+# gitignore contributors: remember to update Global/Xcode.gitignore, Objective-C.gitignore & Swift.gitignore
```
Comment (romoore) [inline .gitignore]: Is this comment necessary? I don't see those files.

File: test_host/BUILD [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    name = "Main",
+    testonly = 1,
+    srcs = ["main.m"],
+)	
+
+# Default test host for unit tests.	
+ios_application(
+    name = "test_host",
+    testonly = 1,
+    bundle_id = "com.google.skylark.testing.DefaultTestHost",
```
Comment (romoore) [inline test_host/BUILD]: Name the bundle to match this project?

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+*.dSYM.zip
+*.dSYM
+
+# CocoaPods
+#
+# We recommend against adding the Pods directory to your .gitignore. However
+# you should judge for yourself, the pros and cons are mentioned at:
+# https://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control
+#
+Pods/
```
Comment (romoore) [inline .gitignore]: If this is recommended against in our own comment, why are we adding it?

File: .gitignore [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+fastlane/report.xml
+fastlane/screenshots
+
+# Code Injection
+#
+# After new code Injection tools there's a generated folder /iOSInjectionProject
+# https://github.com/johnno1962/injectionforxcode
+
+iOSInjectionProject/
```
Comment (romoore) [inline .gitignore]: Can this be removed? I don't see it being used by this test host.

File: .travis.yml [hunk 6]
```diff
@@ -0,0 +1,11 @@
+language: objective-c
+os: osx
+osx_image:
+  - xcode11.3
+  - xcode11.2
+  - xcode10.3
+before_install:
+  - wget -O bazelisk "https://github.com/bazelbuild/bazelisk/releases/download/v1.2.1/bazelisk-darwin-amd64"
+  - chmod +x "bazelisk"
```
Comment (romoore) [inline .travis.yml:13]: Should the checksum be validated after download?

**Review by romoore — COMMENTED** [review]
I don't understand the .gitignore entries - is this test host going to support Carthage and Fast Lane for distribution? I wouldn't expect .gitignore to propagate into clients who depend on it as a module.

**Comment by jverkoey** [comment]
> I don't understand the .gitignore entries - is this test host going to support Carthage and Fast Lane for distribution? I wouldn't expect .gitignore to propagate into clients who depend on it as a module.

They were cargo-culted in. I've removed the unnecessary ones.

File: test_host/BUILD (hunk 3 above)
Comment (jverkoey) [reply test_host/BUILD]: Done.

**Review by romoore — APPROVED** [review]
LGTM pending resolution of bazelisk checksum validation.

File: .travis.yml (hunk 6 above)
Comment (jverkoey) [reply .travis.yml:13]: Added a check.

Comment (romoore) [reply .travis.yml:13]: Thanks!

**Pull request merged**